conf/application.conf

```diff
@@ -57,7 +57,8 @@ auditing {
 mongodb {
   encryption.key = "QmFyMTIzNDVCYXIxMjM0NQ=="
   uri = "mongodb://localhost:27017/"${appName}
-  timeToLive = 28days
+  timeToLive = 30days
```

Comment:
I still feel this should be 15 minutes as the mongo repository using the value right now is only for session data.